src/smoothscroll.js

```diff
@@ -245,13 +245,13 @@ function polyfill() {
     if (shouldBailOut(arguments[0]) === true) {
       original.scroll.call(
         w,
-        arguments[0].left !== undefined
+        arguments[0] !== null && arguments[0].left !== undefined
           ? arguments[0].left
-          : typeof arguments[0] !== 'object'
+          : arguments[0] === null || typeof arguments[0] !== 'object'
```

Comment:
Update: Checking against `null` and allowing it to be passed through to the original scroll function. If this isn't desirable I can revert it.